index.js

```diff
@@ -339,6 +352,20 @@ function addTest(testTitle, testCaseId, buildUrl) {
   }
 }
 
+function getAttachmentPath(attachmentPath, testCaseId) {
+  const files = getFiles(attachmentPath)
```

Comment:
it is possible that multiple tests have the same testCaseId. 

Is it possible to get screenshot name directly from test object? It would reduce complexity a lot.

Reply:
Well, we could use full test name and default path without subfolders, but I'm pretty sure that it all fully depends on user's testing framework settings. 
Ex. : Cypress got it's own subfolder for each spec file.

But still full test name can be used for screenshot search to make usage of "shared" testCaseId possible.

Reply:
Most frameworks can generate full path to screenshots, not sure if cypress can do it.
That would help a lot